src/tools/opensc-tool.c

```diff
@@ -590,6 +593,7 @@ static int list_algorithms(void)
 		{ SC_ALGORITHM_RSA_HASH_SHA224,    "sha224"    },
 		{ 0, NULL }
 	};
+// clang-format on
```

Comment:
clang-format suggested the following change:


```
-// clang-format on
+	// clang-format on
```

<!-- identifier: clang-format -->
<!-- File: src/tools/opensc-tool.c -->